app/src/main/java/org/mozilla/fenix/utils/Undo.kt

```diff
@@ -40,29 +45,60 @@ fun CoroutineScope.allowUndo(
     // writing a volatile variable.
     val requestedUndo = AtomicBoolean(false)
 
-    // Launch an indefinite snackbar.
-    val snackbar = FenixSnackbar
-        .make(view, FenixSnackbar.LENGTH_INDEFINITE)
-        .setText(message)
-        .setAnchorView(anchorView)
-        .setAction(undoActionTitle) {
-            requestedUndo.set(true)
+    // Launch an alert dialog for undo- for accessibility users
```

Comment:
This is the kind of comment that could age poorly.  If we explain the accessibility problem down where we're conditioning on accessibility mode, hopefully it will be removed if that ever goes out of date.  However, it's not unreasonable to think that some other chunk of code could someday call `showUndoDialog` for a use case that has nothing to do with accessibility.  At that point, this comment will have rotted unintentionally.  I would remove this one entirely.

Commenting is _really hard._